microprot/scripts/clustering/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#============================================================
+# Script clusters faa file using MMseqs in three steps:
+# - creates database
+# - uses linclust for initial clustering
+# - uses clust for final clustering
+
+# Usage:
+# - change input parameters
+# - uncomment appropriate commands
```

Comment:
that's not an acceptable structure of a script.
instead, could you create an `if` and read CLI flag to determine what to do?

Reply:
totally agree. I added CLI support